pkg/scheduler/conf/scheduler_conf.go

```diff
@@ -37,6 +46,8 @@ type PluginOption struct {
 	EnabledJobOrder *bool `yaml:"enableJobOrder"`
 	// EnabledJobReady defines whether jobReadyFn is enabled
 	EnabledJobReady *bool `yaml:"enableJobReady"`
+	// EnabledJobBackfillReady defines whether jobBackfillReadyFn is enabled
+	EnabledJobBackfillReady *bool `yaml:"enableJobBackfillReady"`
```

Comment:
BackfillReady or BackfillEligable?

Reply:
BackfillEligable checks whether a job can be backfilled (e.g. all task pending)

EnabledJobBackfillReady is removed

Reply:
if so, why only provide one configuration?